pkg/prommetrics/prommetrics.go

```diff
@@ -121,6 +132,7 @@ func init() {
 
 	metrics.Registry.MustRegister(triggerTotalsGaugeVec)
 	metrics.Registry.MustRegister(crdTotalsGaugeVec)
+	metrics.Registry.MustRegister(buildInfo)
```

Comment:
Actually, can we just call the new function here and completely skip the call in main.go?